### ISSUE TITLE
No such table (Ionic 2 framework)

### ISSUE THREAD

**Issue by michaeljcoyne** [brody2consult/cordova-sqlite-ext#53]
This is my code.

```
this.database = new SQLite();  
this.database.openDatabase({name: "database.sqlite", location: "default", createFromLocation: 1})
```
If I change the name of the database the console says the database is opened even though it doesn't exist.

**Comment by iursevla** [comment]
I think that is expected behaviour. If there's no database it tries to create a new one, so after creation it exists.

**Comment by brody2consult** [comment]
Thanks @iursevla, that is correct. Is this not clear from the documentation?

**Comment by michaeljcoyne** [comment]
Yes but how can I open the actual database and tell it has opened it? Because it is saying the table doesn't exist, when the console says the database I specified (residing in the www) has been opened ok.  (I am using ionic 2)

**Comment by brody2consult** [comment]
When you open the database you can check the sqlite_master table. It will show you which tables are in the database.
